handbook/engineering/core-services/index.md

```diff
@@ -17,10 +17,12 @@ Go, Postgres, Redis, Docker, Kubernetes.
 
 ### Weekly check-ins
 
-Everyone writes a weekly check-in at the end of each Friday in the active tracking issue of the on-going iteration. These check-ins describe the work done in the previous week as well as what's to come in the next week.
+Everyone writes a weekly check-in in the active tracking issue of the on-going iteration. These check-ins describe the work done in the previous week as well as what's to come in the next week.
 
 This process helps each teammate reflect and plan each week proactively and enables other parts of the organization to have visibility into on-going progress.
 
+Both Friday end of day and Monday beginning of the day are appropriate times to write this check-in.
```

Comment:
I think this might be easier to understand:

```suggestion
Appropriate times to write this check-in are end of day on Friday or beginning of day on Monday.
```

But that comes from a mixture of gut feeling and personal taste. Feel free to ignore.